transaction/atomic.go

```diff
@@ -54,11 +54,8 @@ func getAccountDetails(hezClient client.HermezClient, address string,
 	return
 }
 
-// AtomicTransfer perform token or ETH transfers in a pool of transactions
-func AtomicTransfer(hezClient client.HermezClient, ethereumChainID int,
-	txs []AtomicTxItem) (serverResponse string, err error) {
-	atomicGroup := api.AtomicGroup{}
-
+// CreateFullTxs turn the basic information in a PoolL2Tx
```

Comment:
Not convinced of the function name although I don't have a better idea right now. However I think it's worth to explain further what's going on in the function documentation:

- Set metadata: TxID, Type
- set fields based on current state:  set nonce, fromIdx, toidx, ...
- Link txs: set the Rq* fields